Common/Gfx/Canvas.h

```diff
@@ -74,8 +76,10 @@ class Canvas
 	void DrawMaskedBitmap(Gdiplus::Bitmap* bitmap, Gdiplus::Bitmap* maskBitmap, const Gdiplus::Rect& dstRect,
 		const Gdiplus::Rect& srcRect, const Gdiplus::Rect& srcRect2);
 
-	void FillRectangle(Gdiplus::Rect& rect, const Gdiplus::SolidBrush& brush);
+	void DrawGeometry(const Shape& shape, D2D1_MATRIX_3X2_F& transform);
```

Comment:
Always use `const` if you can because it tells the reader that the variable will not be modified.